### ISSUE TITLE
Health of this repo, how to enhance?

### ISSUE THREAD

**Issue by mrienstra** [jqlang/jq#2495]
It looks like `jq` could use some support.

1. 159 commits since last release (Nov 2018): https://github.com/stedolan/jq/compare/jq-1.6...master
2. 15 PRs opened in the last 6 months: 12 PRs still open, 3 PRs closed, 0 merged
3. 18 PRs opened between 6-12 months ago: 3 PRs still open, 6 PRs closed, 9 merged
4. 46 issues opened in the last 6 months: 31 still open, 15 closed (closed issues appear to be overwhelming support requests closed after someone shared a solution to the problem)
5. [Issue requesting sponsor button](https://github.com/stedolan/jq/issues/2346) (so people can financially support this project) has no official response

Seems like this project could really use collaborators / maintainers, or transfer to an org.

I'm not qualified, but I'm happy to help research and/or reach out to potential candidates.

@stedolan, @nicowilliams, @dtolnay, @wtlangford, @muhmuhten, @pkoppstein, @agordon, @orzen, @DRMacIver, @stagrlee, @jankatins, @dgryski, @adoxa, @stroan
(sorry for the mass @ mention, if your name is here, it's because you've made 5+ commits to this repo, unsubscribe if you don't want to receive further notifications)

---
PS: gotta give props to @wader, who has responded to many of the issues that I looked at (those closed in the last 6 months).

**Comment by wader** [comment]
@mrienstra thanks 😊 doing what i can to help out. hope we can work out something to continue jq:s development. my feeling is that it's more relevant than ever as there is so much json and [my own experiments](https://github.com/wader/fq) has convinced me it can be used for much more

**Comment by mrienstra** [comment]
Update, paraphrased from an email from @nicowilliams:
>### Priorities
>
>- Get the Windows build fixed and tests passing on Windows
>
>- Add GitHub Actions for the following builds:
>
>    - Linux 64-bit on x64
>    - OS X (on both architectures?)
>    - Windows 64-bit
>    - maybe Linux on other architectures
>
>### After the above is done
>- Make a release
>- New maintainer

@nicowilliams also said:
>If someone curates a few PRs, I'll do a quick review and merge them

**Comment by dave-r12** [comment]
Hey @mrienstra @wader @nicowilliams I opened a PR to build/test with GitHub actions. If anyone has time, could you review? I encountered 2 issues.

1. The MacOS build was failing with this error:
> posix.c:94:3: error: implicit declaration of function 'onig_end' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  onig_end();
  ^

I was able to fix it by updating oniguruma to 6.7.1 as they fixed this with a commit here:
https://github.com/kkos/oniguruma/commit/5a24a49d710a9e3bb8ff11d12e1eae5a9f9df40c

All tests are passing so thinking this update should be OK. Thoughts?


2. The Windows build uses MSYS2-MINGW64. The build succeeds. However, a single test is failing:
> Test #334: '[range(-52;52;1)] as $powers | [$powers[]|pow(2;.)|log2] == $powers' at line number 1568
*** Expected true, but got false for test at line number 1570: [range(-52;52;1)] as $powers | [$powers[]|pow(2;.)|log2] == $powers

This one is weird? Apparently the log2 functionon Cygwin(?) isn't able to calculate exact results for powers of 2, which seems strange. This is the results of running that tests on Windows:
```
# ./jq.exe '[range(-52;52;1)] as $powers | [$powers[]|pow(2;.)|log2]'
null
[
  -52,
  -51.00000000000001,
  -50,
  -49,
  -48,
  -47.00000000000001,
  -46,
  -45,
  -44,
  -43,
  -42,
  -41,
  -40,
  -39.00000000000001,
  -38,
  -37,
  -36,
  -35,
  -34,
  -33,
  -32,
```

I don't believe this is related to any code in `jq` as I get the similar result with this simple call:
```
log2(pow(2,51))
```

Not fully sure yet how to proceed on that.

**Comment by itchyny** [comment]
Since [jq 1.7](https://github.com/jqlang/jq/releases/tag/jq-1.7) has been released, I'm closing this issue.
